scripts/import-weekly-managed.sh

```
@@ -0,0 +1,497 @@
+  log "ClusterClaim ${CLUSTERCLAIM_NAME} already exists. Exiting."
+  exit 0
+fi
+
+INIT_POOL_SIZE=$(
+  oc get clusterpool.hive \
+    -n "${CLUSTERPOOL_TARGET_NAMESPACE}" \
+    "${CLUSTERPOOL_NAME}" \
+    -o jsonpath='{.spec.size}'
+)
+if ((INIT_POOL_SIZE < 1)); then
+  log "ClusterPool "${CLUSTERPOOL_NAME}" does not meet the minimum of 1. Increasing the size of the pool."
```

Shellcheck (0.11.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)